.github/workflows/pull_request.yml

```diff
@@ -24,7 +24,10 @@ jobs:
         run: |
           ln -snf $PWD/.linters/cpp/hooks/pre-commit.sh $PWD/.linters/cpp/pre-commit.sh
           .linters/cpp/pre-commit.sh $(git --no-pager diff --diff-filter=d --name-only HEAD^ HEAD)
-          git diff -U0 --no-color --staged | "$(dirname "$(which clang-format)")"/../share/clang/clang-format-10/clang-format-diff.py -p1
+      - name: Format check
+        run: |
+          res=$(git diff -U0 --no-color HEAD^ | /usr/share/clang/clang-format-10/clang-format-diff.py -p1)
+          [[ ! -z "$res" ]] && exit 1 || true
```

Comment:
Why need this check? If previous command failed directly.

Reply:
clang-format-diff always returns the diff changes not the state.